app.js

```diff
@@ -207,3 +213,21 @@ linkButton.forEach((par, index) => {
     });
   });
 });
+
+/* Validate form email field by restricting to small letters only */
+
+const form = document.querySelector('#contact-form');
+const mail = form.elements.email;
+const container = document.querySelector('#field');
+const msg = document.createElement('small');
+
+form.addEventListener('submit', (event) => {
+  const emailRegex = (/^[a-z]+$/);
+  if (!emailRegex.test(mail.value)) {
+    event.preventDefault();
+    msg.innerHTML = 'Please enter lowercase letters only in your email.';
+    mail.style.border = '1px solid red';
+    container.appendChild(msg);
+  }
```

Comment:
- [ ] According to the [project requirements ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#project-requirements)

> You need to implement a simple validation:
> 
> - The content of the email field has to be in lower case.


However when I try to enter a correct email with all lowercase it still show an error message `"Please enter lowercase letters only in your email."` Please make sure to implement a validation to check for lowercase i.e. `email === email.toLowerCase()`

 
_**Your contact form:**_

<img width="513" alt="Screenshot 2023-04-21 at 9 00 54 PM" src="https://user-images.githubusercontent.com/44798044/233684736-b86f1fb3-566b-48a0-83ba-4c9786554780.png">